### ISSUE TITLE
[IMPROVEMENTS]: 

### ISSUE THREAD

**Issue by Z0SC** [typical-developers/oaklands-feedback#250]
### Roblox Username

Z3SC

### What do you think can have quality-of-life improvements?

Hotkeys for Equipment slots

### How do you think things can be improved?

Similar to how the normal backpack / inventory allows the player to press the numbers 1-9 to automatically equip / unequip items, this addition will use the keybinds SHIFT + NUMBER (!, @, #, $, ect.) to ENABLE and DISABLE equipment quickly and easily.
I made this quick mockup of what I think it would look like if implemented into Oaklands.
![image](https://github.com/typical-developers/oaklands-feedback/assets/161517180/d2343fde-0642-4337-a702-d86cd4237978)

**Comment by HooferDevelops** [comment]
I don't think Shift is the best idea as it's used for sprinting, but Ctrl might work great!

**Comment by hokies4life06** [comment]
Will there be a way for console players to do something similar to this as well?

**Comment by HooferDevelops** [comment]
Controller has such a limited availability for binds, I'm not sure if it is possible. I didn't even think I would be able to do it for PC.

**Comment by Z0SC** [comment]
Yeah, if you guys can think of better hotkey options that would be great! I just hate the fact that the highest tier Scooba tank requires you to enable and disable it to use its function effectively.

**Comment by BuzzingSniper38** [comment]
Unless y'all find a way I don't think it would be possible to do such thing on controller this is a cool idea if you do find a way for PC Tho, the only way I know for controller is if they have an Xbox elite controller as they have additional buttons on them and you can configure them to any PC controls
